vehicle-python/vehicle/command_line.py

```diff
@@ -37,7 +42,12 @@ def call_vehicle_to_generate_loss_json(
         args = [
             "compile",
             "--target",
+            # use the string below to specify which translation is used. the options are:
+            # LossFunction (uses DL2 by default), LossFunction-DL2, LossFunction-Godel, LossFunction-Lukasiewicz,
+            #  LossFunction-Product, LossFunction-STL (in progress, not currently working)
             "LossFunction",
+            # for experiments,
+            # os.getenv('DLV'),
```

Comment:
What does this commented out line do? Should it be removed?

Reply:
That was for experiment setup where different type of loss function was passed to command line. Will remove from here, not needed in general.